package.json

```diff
@@ -8,10 +8,37 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "build": "webpack -d --watch",
+    "start": "node ./server/server.js",
+    "devserver": "nodemon ./server/server.js",
+    "test": "jest"
+  },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "<rootDir>client/setupTests.js"
+    ]
   },
   "dependencies": {
+    "@babel/core": "^7.5.0",
+    "@babel/polyfill": "^7.4.4",
+    "@babel/preset-env": "^7.5.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^6.26.3",
```

Comment:
It seems odd that you're using `babel-core` _and_ `@babel/core`. Is this intentional?